.github/workflows/check-outdated-content.yaml

```
@@ -40,15 +41,20 @@ jobs:
         
         # Set L10n directory and code
         case "${L10N_BRANCH}" in
-          dev-ko)          
-          L10N_DIR="content/ko/"
-          L10N_CODE="ko"
+          dev-pt)
+          L10N_DIR="content/pt-br/"
+          L10N_CODE="pt"
+          ;;
+
+          dev-zh)
+          L10N_DIR="content/zh-cn/"
+          L10N_CODE="zh"
+          ;;
+
+          *)
```

Hmm, my suggestion is as follows:

```suggestion
          dev-pt)
          L10N_DIR="content/pt-br/"
          L10N_CODE="pt-br"
          ;;

          dev-zh)
          L10N_DIR="content/zh-cn/"
          L10N_CODE="zh-cn"
          ;;

          dev-tw)
          L10N_DIR="content/zh-tw/"
          L10N_CODE="zh-tw"
          ;;

          *)
```

By this, we can set `L10N_CODE` as `zh-cn` for `dev-zh` branch, and `zh-tw` for `dev-tw` branch.
IMO this will make the [name of GitHub workflow step](https://github.com/electrocucaracha/glossary/blob/enable_gh_check_versions/.github/workflows/check-outdated-content.yaml#L167) and the [log](https://github.com/electrocucaracha/glossary/blob/enable_gh_check_versions/.github/workflows/check-outdated-content.yaml#L109) more clear and easily identifiable.. 😊

---

Related to the [name of GitHub workflow step](https://github.com/electrocucaracha/glossary/blob/enable_gh_check_versions/.github/workflows/check-outdated-content.yaml#L167), this is the directory where we store diff results between files. 

This is the `${OUTPUT_DIR}` 😅 
```
OUTPUT_DIR="./outdated"
```

I think it can be classifiable by L10N_CODE in the name section below.
```
    - name: Upload output
      uses: actions/upload-artifact@v3
      with:
        name: ${{ env.L10N_CODE }}-outdated-checking-result
        path: ${{ env.OUTPUT_DIR }}/
```


The [log](https://github.com/electrocucaracha/glossary/blob/enable_gh_check_versions/.github/workflows/check-outdated-content.yaml#L109) part is redundant, so we need to remove that area.